src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        return Color.values()[getRandom().nextInt(Color.values().length)].toString();
```

Comment:
The `getRandomColor` method returns a `String`, but it might be more appropriate to return a `Color` type directly from the `Color` enum. This would maintain type safety and consistency across your application.